client/src/utils/api.js

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+const api = axios.create({
+  url: 'http://localhost:3001',
```

Comment:
we don't need to specify the URL as localhost:3001 since we have proxy described in package.json

Reply:
Oh perfect, I'll remove line 4 and keep the Content type?